db/migrate/20161231030437_create_days.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDays < ActiveRecord::Migration[5.0]
+  def change
+    create_table :days do |t|
+      t.references :calendar
+      t.string :image_url, null: false
+      t.string :location, null: false
```

Comment:
@LevytsRoman - should location be null false? Do we want a location to be on every day at every hour, including for Twitter handle or do we just want the location to appear at the end of the day with the final image (img 25)?